Tests/UITextViewSpecs.swift

```diff
@@ -0,0 +1,182 @@
+import Quick
+import Nimble
+import ViewConfigurator
+
+class UITextViewSpecs: QuickSpec {
+    
+    override func spec() {
+        it("can set text") {
+            let text = "Testing setting the text"
+            let testView: UITextView = .build { set in
+                set.text(text)
+            }
+            expect(testView.text).to(equal(text))
+        }
+        
+        
+        it("can set font") {
+            let font = UIFont(name: "BradleyHandITCTT-Bold", size: 14)
+            let testView: UITextView = .build { set in
+                set.font(font!)
+            }
+            expect(testView.font).to(equal(font))
+        }
+        
+        it("can set text colour") {
+            let color = UIColor.red
+            let testView: UITextView = .build { set in
+                set.textColor(color)
+            }
+            expect(testView.textColor).to(equal(color))
+        }
+        
+        it("can set text allignment") {
+            let textAlignment = NSTextAlignment.right
+            let testView: UITextView = .build { set in
+                set.textAlignment(textAlignment)
+            }
+            expect(testView.textAlignment).to(equal(textAlignment))
+        }
+        
+        it("can set selected Range") {
+            let selectedRange = NSMakeRange(6, 0)
+            let testView: UITextView = .build { set in
+                set.text("random text").selectedRange(selectedRange)
+            }
+            expect(testView.selectedRange).to(equal(selectedRange))
+        }
+        
+        it("can set isEditable") {
+            let editable = true
+            let testView: UITextView = .build { set in
+                set.isEditable(editable)
+            }
+            expect(testView.isEditable).to(equal(editable))
+        }
+        
+        it("can set isSelectable") {
+            let selectable = true
+            let testView: UITextView = .build { set in
```

Comment:
The default Value for Selectable is true, therefor this test would also succeed if nothing would be configured